Source/WebCore/rendering/RenderFlexibleBox.h

```diff
@@ -47,6 +48,8 @@ class RenderFlexibleBox : public RenderBlock {
     RenderFlexibleBox(Document&, RenderStyle&&);
     virtual ~RenderFlexibleBox();
 
+    using Direction = WritingMode;
```

Comment:
There's now a BlockFlowDirection enum since the sideways writing mode refactoring.

WritingMode now corresponds to literally the `writing-mode` CSS values (e.g. horizontal-tb, etc.)